week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,22 +22,21 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
+  letters.sort();
+  let sortedLetters = letters;
   return sortedLetters;
 }
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
-
+  numbers.length = 5;
+  let sliced = numbers;
   return sliced;
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
-
+  let index;
+  let element = arr[3];
```

Comment:
If you didn't change the line about this could be written as

`let element = arr[index];`

Since you can pass in a variable to the array to get the value